src/pages/[language]/jobs/index.vue

```diff
@@ -7,22 +7,10 @@
       :image="data.page.headerIllustration"
     />
     <div class="grid">
-      <ul
-        class="page-jobs__overview"
+      <jobs-list
         v-if="data.items.length"
-      >
-        <li
-          v-for="item in data.items"
-          :key="item.slug"
-        >
-          <jobs-excerpt
-            :title="item.title"
-            :label="item.callToActionLabel"
-            :image="item.jobImage"
-            :slug="item.slug"
-          />
-        </li>
-      </ul>
+        :jobs="data.items"
+      />
```

Comment:
The jobs page can be made a landing page now

Reply:
I'll create a ticket for that